### DISCUSSION TITLE
Is there any interest in Windows binaries?

### DISCUSSION THREAD

**Discussion by atc0005** [atc0005/check-vmware#178] [Poll]
## What

Originally posted to GH-160:

> Build times are ever increasing, as is the total size of all compiled binaries. I'm considering providing only Linux x86 and x64 binaries as part of future releases.
> 
> If there is interest, I'll continue to provide both sets of binaries for each release, otherwise I'll plan to drop the Windows binaries with the next release.
> 
> Building Windows binaries locally via `make windows` should continue to work as expected.

Windows binaries were dropped from the [v0.13.0 release](https://github.com/atc0005/check-vmware/releases/tag/v0.13.0), but can be brought back if there is sufficient interest.

## Local builds

Support for building Windows binaries locally is possible via:

- `env GOOS=windows GOARCH=386 go build ./cmd/PLUGIN`
- `env GOOS=windows GOARCH=amd64 go build ./cmd/PLUGIN`
- `Makefile` builds
  - `make windows`
    - 32-bit and 64-bit Windows binaries
  - `make all`
    - 32-bit and 64-bit Linux & Windows binaries

## How to provide feedback

- Upvote for "yes", I want Windows binaries.
- Downvote for "no", I do not use or have interest in Windows binaries.

**Comment by tubby1981** [comment]
We are using debian / Ubuntu on our nagios servers. No interest in Windows :-)

**Reply by atc0005** [reply]
Thanks for the feedback!
